cmd/common_helpers.go

```diff
@@ -136,6 +152,35 @@ func printProjectsOut(cmd *cobra.Command, projects ...*gitlab.Project) {
 }
 
 func printGroupMembersOut(cmd *cobra.Command, members ...*gitlab.GroupMember) {
+	if len(members) == 0 {
```

Comment:
* This condition should only apply during the `default` case. It is ok to print the default nothing format of YAML and JSON to users.
* Let's also make it simple by just saying _"The get command returned no results."_,  if get command result length returned is zero.

Some other notes..

* Please keep the column length to 80 when typing long code.
* Use `%s` instead of `%v` when formatting a string.